src/app/component/Slider/Slider.component.js

```diff
@@ -21,6 +21,12 @@ import './Slider.style';
 export const ANIMATION_DURATION = 300;
 export const ACTIVE_SLIDE_PERCENT = 0.1;
 
+import {
+    TABLET_WIDTH,
+    DESKTOP_WIDTH
+} from 'Component/SliderWidget/SliderWidget.component';
```

Comment:
Imports come before exports.

Reply:
That is true, while you should still prefer putting path imports after aliased imports:

```js
import {
    TABLET_WIDTH,
    DESKTOP_WIDTH
} from 'Component/SliderWidget/SliderWidget.component';

import './Slider.style';
```